### DISCUSSION TITLE
Would be great if a NavigationItem within the navigationItems() for a panel could support actions

### DISCUSSION THREAD

**Discussion by hotsaucejake** [filamentphp/filament#13326] [Ideas]
I'm building a non-authenticated panel (irrelevant).  My use case would be to have a Register and Login action button that worked as a modal within the panels navigation.  I haven't explored using a render hook yet for this, but that will be my next step.  However, I think this could be a useful feature.

**Comment by juliancc** [comment]
I second this. For example. I was looking for a way to do something like this:
<img width="290" alt="image" src="https://github.com/user-attachments/assets/e08c5d10-4459-48c6-a8c8-a24510ac9c21" />


Where you can add "quick add" buttons to sub menus
